apps/earn-protocol/hooks/use-timeframes.ts

```
@@ -0,0 +1,62 @@
+    if (!chartData) {
+      return {} as TimeframesItem
+    }
+
+    return Object.keys(chartData.data).reduce<TimeframesItem>(
+      (acc, key) => {
+        const keyTyped = key as keyof ChartsDataTimeframes
+
+        if (keyTyped === '7d') {
+          // we always want to show 7d
+          return {
+            ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)